presto-docs/src/main/sphinx/functions/sysinfo.rst

```diff
@@ -0,0 +1,14 @@
+============================
+System Information Functions
+============================
+
+Functions providing information about the query execution environment.
+
+String Functions
+----------------
+
+.. function:: current_user -> varchar
+
+    Returns the current user running the query.
+
+    .. note:: This is a SQL-standard function and does not use parenthesis.
```

Comment:
Technically, it's not a "function"

Reply:
I found some references in the sql standard that classify it as a value specification, would this be the correct term to use?